src/internal/m365/collection/exchange/mail_backup.go

```diff
@@ -29,6 +29,26 @@ func (h mailBackupHandler) itemHandler() itemGetterSerializer {
 	return h.ac
 }
 
+func (h mailBackupHandler) folderGetter() containerGetter {
+	return h.ac
+}
+
+func (h mailBackupHandler) previewIncludeFolders() []string {
+	return []string{
+		"inbox",
+	}
+}
+
+func (h mailBackupHandler) previewExcludeFolders() []string {
+	return []string{
+		"drafts",
```

Comment:
not sure if we want to adjust this set of excludes for preview backups. It just seemed like things that wouldn't be as important to the user